server/GRPC_TA2_TA3/relay.js

```diff
@@ -0,0 +1,58 @@
+// Javascript Imports
+const path = require('path');
+const grpc = require('grpc');
+const WebSocketServer = require('ws').Server;
+
+// Environment variables for GRPC
+const GRPC_PORT = process.env.GRPC_PORT;
+const GRPC_HOST = process.env.GRPC_HOST;
+const GRPC_CONN_URL = GRPC_HOST + ':' + GRPC_PORT;
+
+// Websocket defs
+let wss = new WebSocketServer({port: 8888});
+
+// Loading proto files
+let core = grpc.load(path.join(__dirname, 'ta3ta2_api', 'core.proto'));
+let data_ext = grpc.load(path.join(__dirname, 'ta3ta2_api', 'data_ext.proto'));
+let dataflow_ext = grpc.load(path.join(__dirname, 'ta3ta2_api', 'dataflow_ext.proto'));
+
+// Creating the GRPC Connections
+let connections = [];
+connections.push( {stub: new core.Core(GRPC_CONN_URL, grpc.credentials.createInsecure()), module: core} );
```

Comment:
nitpicks: These three lines are a bit too long. Consider placing objects on multiple lines.
```js
connections.push({
  stub: new core.Core(GRPC_CONN_URL, grpc.credentials.createInsecure()),
  module: core
});
```
There is no need for spaces between round bracket and curly bracket.